README.md

```diff
@@ -257,3 +256,118 @@ We can now scale Kibana and Fluentd back up to their previous state.  Since Flue
 was shut down and allowed to push its remaining records to ES in the previous
 steps it can now pick back up from where it left off with no loss of logs -- so long
 as the log files that were not read in are still available on the node.
+
+## EFK Health
+
+In order to determine that your EFK deployment is currently healthy and running,
```

Comment:
Determining the health of an EFK deployment and if it is running can be assessed by running the [e2e tests](hack/testing/).  Additionally, you can do the following: